### PR TITLE
CFY-4357-Bootstrap-plugins-upload-documentation-formatting

### PR DESCRIPTION
**Pull request by max-orlov**

### CODE REVIEW

File: content/blueprints/spec-upload-resources.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+draft: false
+weight: 1900
+
+---
+
+# Declaration
+The upload_resources section in the DSL is a dictionary comprises two subsections:
+
+plugin_resources: designated for uploading plugins to the manager.
+dsl_resources: designated for uploading resources needed for the parsing process of blueprints.
```
Comment (nir0s) [inline content/blueprints/spec-upload-resources.md]: the process of parsing blueprints

File: content/blueprints/spec-upload-resources.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# Definition
+Keyname	            |       Required	|   Type	    | Description
+--------------------|-------------------|---------------|-------------------------
+plugin_resources    | no	            |   list	    | A list of plugins (URLs or local paths) to be uploaded to the manager.
+dsl_resources	    | no	            |   dict	    | A list of dictionaries each comprises a source_path and destination_path for each dsl_resource.
+source_path	        | yes	            |   string	    | The source path for the dsl resource.
+destination_path    | yes	            |   string	    | A relative destination path for the resource (relative to the file server home dir).
+
+{{% gsNote title="Note" %}}
+All plugins uploaded to the manager should be in wgn format.
```
Comment (nir0s) [inline content/blueprints/spec-upload-resources.md]: please provide a reference link to content/plugins/using-plugins.md

File: content/blueprints/spec-upload-resources.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+# Examples
+ 
+upload_resources:
+    plugin_resources: 
+     - 'http://www.my-plugin.com/path/to/plugin.wgn'
+    dsl_resources: 
+     - 'source_path': 'http://www.my-plugin.com/path/to/plugin.yaml'
+       'destination_path': '/path/to/local/plugin.yaml'
+ 
+
```
Comment (nir0s) [inline content/blueprints/spec-upload-resources.md]: I think this section requires some more explanation. It isn't clear which resources could be uploaded and why.

File: content/blueprints/spec-upload-resources.md (hunk 2 above)
Comment (max-orlov) [inline content/blueprints/spec-upload-resources.md]: do you mean /blueprints/spec-plugins.md? 
there is no /plugins/using-plugins.md in the master.

Comment (nir0s) [inline content/blueprints/spec-upload-resources.md]: nope.

File: content/blueprints/spec-upload-resources.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    plugin_resources: 
+     - 'http://www.my-plugin.com/path/to/plugin.wgn'
+    dsl_resources: 
+     - 'source_path': 'http://www.my-plugin.com/path/to/plugin.yaml'
+       'destination_path': '/path/to/local/plugin.yaml'
+{{< /gsHighlight >}}
+
+In this example we can see the upload of 2 different resources:
+
+- In the `plugin_resources` section, the plugin.wgn is being uploaded. This plugin would be used any time this plugin is needed by any blueprint.
```
Comment (nir0s) [inline content/blueprints/spec-upload-resources.md]: will

File: content/blueprints/spec-upload-resources.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+- In the `dsl_resources` section, the yaml file for that plugin is being retrieved from 'http://www.my-plugin.com/path/to/plugin.yaml', and is being uploaded to 
+the manager file server with the relative path of '/path/to/local/plugin.yaml'. This resource would be used anytime the manager parses a blueprint which references
```
Comment (nir0s) [inline content/blueprints/spec-upload-resources.md]: manager's
would -> will

no more comments. merge

**Pull request merged**